LICENSE

```diff
@@ -1,4 +1,4 @@
-Copyright (c) 2012-2022 Fabien Potencier, Dariusz Rumiński
+Copyright (c) 2012+ Fabien Potencier, Dariusz Rumiński
```

Comment:
```suggestion
Copyright (c) 2012 Fabien Potencier, Dariusz Rumiński
```

Perhaps suffices?

Reply:
no clue from legal side. i didn't want to give impression that it was only under copyrights for one year, as we had the range [start, end] before